src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesRecorder.java

```diff
@@ -435,6 +437,34 @@ public String getReferenceJobName() {
         return referenceJobName;
     }
 
+    /**
+     * Sets the reference build id to get the results for the issue difference computatation.
+     *
+     * @param referenceBuildId
+     *         the build id of the reference job
+     */
+    public void setReferenceBuildId(final String referenceBuildId) {
+        if (NO_REFERENCE_BUILD.equals(referenceBuildId)) {
+            this.referenceBuildId = StringUtils.EMPTY;
+        }
+        else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['}' at column 9 should be alone on a line.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=4335332)